packages/mui-utils/src/generateUtilityClass/generateUtilityClass.ts

```diff
@@ -5,11 +5,12 @@ export type GlobalStateSlot =
   | 'checked'
   | 'completed'
   | 'disabled'
-  | 'readOnly'
   | 'error'
   | 'expanded'
   | 'focused'
   | 'focusVisible'
+  | 'open'
```

Comment:
Wouldn't this create breaking changes in Material UI? The class names would be different.

Reply:
No, apparently it's not used anywhere in Material UI.